src/api/routes/core-node-rpc-proxy.ts

```diff
@@ -128,6 +134,22 @@ export const CoreNodeRpcProxyRouter: FastifyPluginAsync<
     }
   }
 
+  /// Checks if we should modify all transaction fee estimations to always use the minimum fee. This
+  /// only happens if there is no fee market i.e. if the last N block tenures have not been full. We
+  /// use a threshold of 90% to determine if a block size dimension is full.
+  async function shouldUseTransactionMinimumFee(): Promise<boolean> {
+    const tenureWindow = parseInt(process.env['STACKS_CORE_FEE_TENURE_FULLNESS_WINDOW'] ?? '5');
+    const averageCosts = await fastify.db.getTenureAverageExecutionCosts(tenureWindow);
```

Comment:
can we move this function call out of this function and pass in the return value to the top level function